PowerPlant.cs

```
@@ -0,0 +1,113 @@
+using System;
+
+namespace LambdaCore
+{
+    public class PowerPlant
+    {
+        private List<Core> cores = new List<Core>();
+        private Core selectedCore = null;
+
+        public void CreateCore(string type, int durability)
+        {
+            char coreName = (char)('A' + cores.Count());
+
+            if (type == "System")
+            {
+                Core core = new Core(TypeCore.SystemCore, coreName.ToString(), durability);
+                cores.Add(core);
+                Console.WriteLine($"Successfully created Core {coreName}!");
+            }
+            else if (type == "Para")
+            {
+                Core core = new Core(TypeCore.ParaCore, coreName.ToString(), durability);
+                cores.Add(core);
+                Console.WriteLine($"Successfully created Core {coreName}!");
```

You can skip the type checking for the core by passing an enum directly instead of a string as a param

public void CreateCore(string type, TypeCore durability)

---

If you want you can also lookup the Dictionary data structure and how it works. You can create a 'mapping' string -> core type and use it here in place of the if / else if statements. if the core type is not present in the dictionary you can throw an exception